src/main/webapp/main-page.html

```diff
@@ -30,23 +31,23 @@ <h4>Share Query</h4>
       <button onmouseover="changeText()" onclick = "copyLink()" > Copy Link </button>
     </div>
     
-    <button id="filter-button " onClick="showFiltersPanel(); filterColumns()"> Filters</button>
-      <div id="filter-box" style="display: none;">
-        <form action="/main-page.html" method="GET">
-          <h1> Filters</h1>
+    <div id="filter-box" style="display: none;">
+      <form action="/main-page.html" method="GET">
+        <h1> Filters</h1>
 
-          <input type="text" id="user_id" placeholder="Enter the user_id">
-          <div style="padding: 10px"></div> 
-          <input type="text" id="device_id" placeholder="Enter the device_id">
-          <div style="padding: 10px"></div>
-          <input type="search" placeholder="Search...">
-          <div style="padding: 10px"></div>
+        <input type="text" id="user_id" placeholder="Enter the user_id">
+        <div style="padding: 10px"></div> 
```

Comment:
Didn't notice these before. What are they for?

Reply:
Padding to space for the user/device id. These have been moved to the toolbar now